src/fprime/util/cli.py

```diff
@@ -65,7 +65,7 @@ def print_info(
         print(
             f"    Available global targets: {' '.join(global_generic_targets.keys())}"
         )
-        print("  ----------------------------------------------------------")
+        print(f'{"  ":-<60}')
```

Comment:
Why not `-` * 60 like above?  I have a hard time reading this one....

Reply:
Now i understand and approve.  Although this is fairly cryptic.

Reply:
(not on you)  This one is on python.  I cannot find the documentation on how that works.

Reply:
Sorry I didn't have time to answer you before.
I preferred this method because I had just learned it on Youtube (https://www.youtube.com/shorts/NHquaDS6XQ0).
I also couldn't find the documentation. I was hoping you knew ahah!
I don't know from which version of Python this was implemented.